packages/devtools_app/lib/src/screens/memory/panes/profile/profile_view.dart

```diff
@@ -318,7 +318,11 @@ class _GCHeapCapacityColumn extends _GCHeapStatsColumn {
 
   @override
   String getDisplayValue(AdaptedProfile dataObject) {
-    return prettyPrintBytes(getValue(dataObject), includeUnit: true)!;
+    return prettyPrintBytes(
+      getValue(dataObject),
+      includeUnit: true,
+      kbFractionDigits: 0,
```

Comment:
@polina-c should these columns actually have KB fraction digits of size 1? This PR changes the default value for this parameter so I added the 0 here to keep the current behavior, but wanted to check if this is the current behavior by accident or if it is intentional.

Reply:
agree that this should probably be changed to 1.

Reply:
done